src/views/Hearing/Sidebar.js

```diff
@@ -35,6 +37,38 @@ class Sidebar extends React.Component {
     );
   }
 
+  getLanguageChanger() {
+    const {hearing, activeLanguage} = this.props;
+    const availableLanguages = { fi: 'Kuuleminen Suomeksi', sv: 'Enkäten på svenska', en: 'Questionnaire in English'};
+    const languageOptionsArray = config.languages.map((lang) => {
+      if (getAttr(hearing.title, lang, {exact: true}) && lang === activeLanguage) {
```

Comment:
Now there are no links in any languages here. The other hearing languages should be available through clicking, even though the highlighted (current) language does not have a link.